app/services/db-export/send-to-s3-bucket.service.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+/**
+ * Sends a file to our AWS S3 bucket
+ * @module SendToS3BucketService
+ */
+
+const { S3Client, PutObjectCommand } = require('@aws-sdk/client-s3')
+const fs = require('fs')
+const S3Config = require('../../../config/s3.config')
+const path = require('path')
+
+/**
+ * Sends a file to our AWS S3 Bucket using the filePath that it receives
+ *
+ * @param {String} filePath A string containing the path of the file to send to the S3 bucket
+ *
+ * @returns {Boolean} True if the file is uploaded successfully and false if not
+ */
+async function go (filePath) {
+  if (!filePath) {
+    global.GlobalNotifier.omfg('ERROR uploading file to S3 bucket. No file path given')
+    return false
+  }
+
+  const bucketName = S3Config.s3.bucket
+  const folderName = 'export'
+  const fileName = path.basename(filePath)
+  const fileContent = fs.readFileSync(filePath)
+
+  const params = {
+    Bucket: bucketName,
+    Key: `${folderName}/${fileName}`,
+    Body: fileContent
+  }
```

Comment:
I think we only use `folderName` when generating the `Key:` value. If so then there is no harm in just using it directly.

```suggestion
  const fileName = path.basename(filePath)
  const fileContent = fs.readFileSync(filePath)

  const params = {
    Bucket: bucketName,
    Key: `export/${fileName}`,
    Body: fileContent
  }
```